### PR TITLE
cfn changes for cudos v5

### PR DESCRIPTION
**Pull request by yprikhodko**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cfn-templates/cid-cfn.yml [hunk 1]
```diff
@@ -143,17 +146,22 @@ Parameters:
...
+    Default: 0.2.31
     Description: A version of CID package
   Suffix:
     Type: String
     Description: Leave Empty. Do not use this Suffix it is not fully supported. For testing purposes only.
     Default: ""
   DeployCUDOSDashboard:
     Type: String
-    Description: Deploy CUDOS Dashboard
+    Description: Set to 'no' to remove deprecated (v4) version of CUDOS Dashboard
```
Comment (vmindru) [inline cfn-templates/cid-cfn.yml:157]: How about forcing the user to choose either, so they don't have to set his to know. e.g. force it with AllowedValues or maybe somehow else ... point being if user selects V5 we should remove V4 maybe or something like this.

Comment (iakov-aws) [reply cfn-templates/cid-cfn.yml:157]: The idea is to allow customer a double run

File: cfn-templates/cid-cfn.yml [hunk 2]
```diff
@@ -143,17 +146,22 @@ Parameters:
   CidVersion:
     Type: String
     MinLength: 5
-    Default: 0.2.30
+    Default: 0.2.31
```
Comment (iakov-aws) [inline cfn-templates/cid-cfn.yml:149]: Better aligning version on a release commit

File: cfn-templates/cid-cfn.yml [hunk 3]
```diff
@@ -51,7 +52,9 @@ Metadata:
       CURBucketPath:
         default: "Path to Cost and Usage report"
       DeployCUDOSDashboard:
-        default: "Deploy CUDOS Dashboard"
+        default: "Deploy CUDOS Dashboard v4 [use for cleanup of old version only]"
```
Comment (iakov-aws) [inline cfn-templates/cid-cfn.yml:55]: suggestion

```
default: "CUDOS Dashboard v4 - Deprecated [set 'no' to delete ]"
```

**Review by iakov-aws — APPROVED** [review]

**Pull request merged**